src/tags/object/AudioUltra/model.js

```diff
@@ -355,6 +355,10 @@ export const AudioModel = types.compose(
 
           if (states.length === 0) {
           // wsRegion.on("update-end", ev=> self.selectRange(ev, wsRegion));
```

Comment:
Let's remove this comment as well. Not needed anymore, the states are handled for all cases.